canal/canal_test.go

```diff
@@ -57,6 +62,22 @@ func (s *canalTestSuite) SetUpSuite(c *C) {
 }
 
 func (s *canalTestSuite) TearDownSuite(c *C) {
+	sc := make(chan os.Signal, 1)
```

Comment:
I am confused with these, why not sleep directly but introduce complex signal and channel?